python/lsst/drp/tasks/assemble_cell_coadd.py

```diff
@@ -323,6 +323,20 @@ def __init__(self, *args, **kwargs):
             self.makeSubtask("scale_zero_point")
 
         self.psf_warper = afwMath.Warper.fromConfig(self.config.psf_warper)
+        if "lanczos" in (warping_kernel_name := self.config.psf_warper.warpingKernelName.lower()):
+            psf_padding = 2 * int(warping_kernel_name.lstrip("lanczos")) - 1
```

Comment:
`lstrip` won't work unless `warping_kernel_name.startswith('lanczos') == True` so you may as well change that condition above.